esbuild.js

```diff
@@ -75,8 +75,12 @@ const options = {
         cssModules(
             {
                 visitor: compileWithCssVariables(),
-                targets: {
-                    chrome: 80 // aligns somewhat to es2020
+                targets: { // only support es2020 browser
```

Comment:
please add these links to the explanation

https://lightningcss.dev/transpilation.html

https://github.com/evanw/esbuild/issues/121#issuecomment-646956379